.gitignore

```diff
@@ -13,6 +13,7 @@ lines-of-code-report.json
 
 # Python
 __pycache__
+.venv/
```

Comment:
Does this need to be added to .dockerignore as well? I don't think it will be picked up by the image as we are selectively copying stuff in, but I think it might be adding to the build context unnecessarily, which would affect build time